x-pack/plugins/security_solution/common/field_maps/field_names.ts

```diff
@@ -17,8 +17,23 @@ export const ALERT_THRESHOLD_RESULT = `${ALERT_NAMESPACE}.threshold_result` as c
 export const ALERT_THRESHOLD_RESULT_COUNT = `${ALERT_THRESHOLD_RESULT}.count` as const;
 export const ALERT_NEW_TERMS = `${ALERT_NAMESPACE}.new_terms` as const;
 export const ALERT_NEW_TERMS_FIELDS = `${ALERT_RULE_PARAMETERS}.new_terms_fields` as const;
-export const ALERT_HOST_CRITICALITY = `${ALERT_NAMESPACE}.host.criticality_level` as const;
-export const ALERT_USER_CRITICALITY = `${ALERT_NAMESPACE}.user.criticality_level` as const;
+/**
+ * @deprecated Use ALERT_HOST_CRITICALITY
```

Comment:
This format will actually link to the other type: 

```suggestion
 * @deprecated Use {@link ALERT_HOST_CRITICALITY}
```